src/sbl_filing_api/entities/models/dto.py

```diff
@@ -32,8 +22,8 @@ class SubmissionDTO(BaseModel):
     validation_json: List[Dict[str, Any]] | None = None
     submission_time: datetime | None = None
     filename: str
-    submitter: SubmitterDTO | None = None
-    accepter: AccepterDTO | None = None
+    submitter: int | None = None
+    accepter: int | None = None
 
```

Comment:
I think we'd want these as the UserAction, not just the id.  We want to make sure the submission and Filing DTO being returned is composed of the full data.